build.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/local/bin/python
+from jinja2 import Template, Environment, FileSystemLoader
+import os
+
+# Capture our current directory
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+
+def render_templates():
+    j2_env = Environment(
+        loader=FileSystemLoader(THIS_DIR),
+        trim_blocks=True
+    )
+
+    context = {
+        "jsrequire": j2_env.get_template("require.js.j2").render(),
+        "jsauth": j2_env.get_template("auth.js.j2").render()
+    }
+
+    files = [
+        "post-events.js",
+        "create-object.js",
+        "create-owner.js"
+    ]
+    for file in files:
+        filename = THIS_DIR + "/dist/" + file
+        print filename
+        with open(filename, 'w') as f:
+            f.write(j2_env.get_template(file + '.j2').render(context))
+
+if __name__ == '__main__':
+    render_templates()
```

Comment:
For the build I would prefer, if possible, a more `nodejs` [solution](https://www.npmjs.com/browse/keyword/jinja). It allows better tooling (already eslint for example) and there seems to be an equivalent of jinja templates for node. Although python is ubiquitous, let's try to focus on node for js projects. What do you think?